### ISSUE TITLE
MXE Building doesn't have CLI interfacing

### ISSUE THREAD

**Issue by ProjectHSI** [raphnet/vspcplay#14]
In Linux builds, the executable generated has an output when ran from the command line, the MXE build for some reason doesn't have this, making it pure guesswork as to what happened when the application doesn't run.

**Comment by raphnet** [comment]
This must have something to do with "console app" vs "gui/windows/WinMain" concept on Windows.

**Comment by ProjectHSI** [comment]
> This must have something to do with "console app" vs "gui/windows/WinMain" concept on Windows.

I think so, I believe the application immediately returns to the command prompt upon opening, I think the configuration for the app needs to change so that's it's not a gui application and a console app (which has a gui)

**Comment by raphnet** [comment]
Yes, by removing the -mwindows build option I believe. But when doing that, I think clicking on the app will open a black terminal window, in addition to the main application window... And I don't know what happens when SDL tries to open a window, does it work if it's not a "gui" app?

I see some potential solutions here, but I don't have a setup to experiment things on Windows at the moment...
https://stackoverflow.com/questions/191842/how-do-i-get-console-output-in-c-with-a-windows-program

**Comment by ProjectHSI** [comment]
> I don't have a setup to experiment things on Windows at the moment...

Well we got ourselves a combo here! I actually *am* a Windows user which is why I care about the MXE build so much.

I don't see an option for -mwindows in the MXE Makefile.

I think it will open a black terminal but that at least will provide stdout, even program made by giant companies can't get this right.
